glowbot/commands.py

```diff
@@ -181,16 +193,26 @@ async def claim(self, ctx: discord.ApplicationContext, hours: Option(
 
                         expiration = cur_expiration + timedelta(hours=grant_value)
 
-                    # Make sure all RCON grants are successful.
-                    result_dict = await self.client.grant_vip(player.player_name, player.steam_id_64, expiration.strftime("%Y-%m-%dT%H:%M:%S%z"))
-                    for rcon, result in result_dict.items():
-                        if result is False:
-                            self.logger.error(f'Problem assigning VIP in `claim` for \"{rcon}\": {result}')
-                            await ctx.respond(f'{ctx.author.mention}: There was a problem on one of the servers assigning your VIP.')
-                            return
+                    message = ""
+                    if cur_expiration >= datetime(year=2200, month=1, day=1, tzinfo=timezone.utc): 
```

Comment:
`cur_expiration` isn't set in all codepaths at this point.
Can just chagne to `expiration`... the added timedelta won't matter in this calculation.